devpack/src/main/java/io/neow3j/devpack/Runtime.java

```diff
@@ -133,6 +133,23 @@ public class Runtime {
     @Instruction(interopService = InteropService.SYSTEM_RUNTIME_GETENTRYSCRIPTHASH)
     public static native Hash160 getEntryScriptHash();
 
+    /**
+     * This method loads the provided bytecode script and executes it with the given call flags and arguments.
+     * <p>
+     * The execution context is limited to read-only ({@link io.neow3j.devpack.constants.CallFlags#ReadOnly})
```

Comment:
Very good to mention this, even though the neo core fails to mention it in their code documentation.